kraken/node_actions/abstract_node_scenarios.py

```diff
@@ -65,6 +65,26 @@ def stop_start_kubelet_scenario(self, instance_kill_count, node, timeout):
         self.node_reboot_scenario(instance_kill_count, node, timeout)
         logging.info("stop_start_kubelet_scenario has been successfully injected!")
 
+
+    # Node scenario to restart the kubelet
+    def restart_kubelet_scenario(self, instance_kill_count, node, timeout):
+        for _ in range(instance_kill_count):
+            try:
+                logging.info("Starting restart_kubelet_scenario injection")
+                logging.info("Restarting the kubelet of the node %s" % (node))
+                runcommand.run("oc debug node/" + node + " -- chroot /host systemctl restart kubelet &")
```

Comment:
Had to put & at end of line to run a detached oc debug. Sometimes the node went NotReady -> Ready while the command was still executing